src/summary_generator.py

```
@@ -1,13 +1,26 @@
-import os 
+
+from pydantic_ai.direct import model_request_sync
+from pydantic_ai.messages import ModelRequest
+
+from scrape_articles import scrape_articles
+
+LLM_PROVIDER = os.getenv("LLM_PROVIDER", "openai")
+LLM_MODEL = os.getenv("LLM_MODEL", "gpt4.1-nano")
 
 
+def llm_request(system_prompt: str, user_prompt: str, *, max_tokens: int, temperature: float, model: str | None = None) -> str:
```

Pylint (3.3.7)

[convention] Line too long (127/100)

(C0301)